src/widgets/forms/forms.component.test.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { cleanup, render, screen } from "@testing-library/react";
+import Forms from "./forms.component";
+import userEvent from "@testing-library/user-event";
+import { navigate } from "@openmrs/esm-framework";
+import { fetchAllForms, fetchPatientEncounters } from "./forms.resource";
+import { of } from "rxjs";
+import {
+  mockForms,
+  mockPatientEncounters
+} from "../../../__mocks__/forms.mock";
+
+const mockNavigate = navigate as jest.Mock;
+const mockFetchAllForms = fetchAllForms as jest.Mock;
+const mockFetchPatientEncounters = fetchPatientEncounters as jest.Mock;
```

Comment:
Matter of opinion, but this is data specific to the forms test, so I don't know why it doesn't just live in the forms test.

Reply:
am confused here, this is `forms.component.test.tsx` file that contains the tests for `forms.component.tsx` maybe you can point me to the best place to have the data specified

Reply:
All I mean is that I don't think `forms.mock.ts` and `visits.mock.ts` really need to exist as their own files — they can just be inlined here at the top of the test file. It's not important though. A matter of opinion.